src/linter/tests/rules/invalid-change-version.test.mjs

```diff
@@ -13,12 +13,15 @@ describe('invalidChangeVersion', () => {
   it('should return an issue if a change version is invalid', () => {
     const issues = invalidChangeVersion({
       ...assertEntry,
-      changes: [...assertEntry.changes, { version: ['v13.9.0', 'REPLACEME'] }],
+      changes: [
+        ...assertEntry.changes,
+        { version: ['v13.9.0', 'INVALID_VERSION'] },
```

Comment:
I would keep the test case to ensure we report as an error if `REPLACEME` is not the only version
```suggestion
        { version: ['v13.9.0', 'REPLACEME'] },
```

Reply:
Got it! Thanks for the clarification